tests/runner.py

```diff
@@ -884,8 +884,8 @@ def get_library(self, name, generated_libs, configure=['sh', './configure'],
 
     emcc_args = self.get_emcc_args()
 
-    hash_input = (str(emcc_args) + ' $ ' + str(env_init)).encode('utf-8')
```

Comment:
this stops using `env_init`, was that intentional? i think that might be a functional change, but I don't remember all the details of how caching works in that code.